### PR TITLE
ci(GHA): Add design-tokens tests to b&tf

### LINKED ISSUE #1969: Run design-token package test suites as part of CI
These are not currently being run as part of CI - this package was recently introduced.

### PR DESCRIPTION
**Pull request by elglup**
## Related issue

closes #1969 

## Overview

Add design-tokens tests to build & test workflows

## Work carried out

- [x] Edited build_and_test_feature.yml
- [x] Edited build_and_test_master.yml

### CODE REVIEW

File: .github/workflows/build_and_test_feature.yml [hunk 1]
```diff
@@ -256,6 +256,30 @@ jobs:
...
+             key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+         - name: Attach workspace
+           uses: actions/download-artifact@v2
+           with:
+             name: dist
+
+         - name: Jest design-tokens
+           run: |
+             tar -xzf dist.tar.gz && mv distil packages/icon-library/dist && mv distdt packages/design-tokens/dist
```
Comment (m7kvqbe1) [inline .github/workflows/build_and_test_feature.yml]: I think you can probably run the tests without needing the built versions of those other packages. 

So this line is maybe redundant?

**Review by m7kvqbe1 — COMMENTED** [review]
Just left a little comment about those `dist` directories.

File: .github/workflows/build_and_test_feature.yml (hunk 1 above)
Comment (elglup) [reply .github/workflows/build_and_test_feature.yml]: Thanks, have removed now

**Review by m7kvqbe1 — APPROVED** [review]
LGTM

**Pull request merged**